core/src/main/java/nl/nn/adapterframework/doc/objects/AMethod.java

```diff
@@ -0,0 +1,71 @@
+package nl.nn.adapterframework.doc.objects;
+
+import java.util.ArrayList;
+
+public class AMethod {
+
+    private String name;
+    private String className;
+    private String originalClassName;
+    private String folderName;
```

Comment:
Moeten className, originalClassName en foldername niet attributen zijn van AClass in plaats van van AMethod?
En dat AMethod dan een verwijzing heeft naar AMethod?

Reply:
className en folderName kunnen inderdaan beter een attribuut zijn van AClass. originalClassName verschilt echter per method, dus blijft het een attribuut van AMethod